### PR TITLE
[WIP] Development Rhythm Series

### PR DESCRIPTION
**Pull request by billkable**
Following articles:

-   Development Flow

-   Development Building Blocks, Part 1

-   Development Building Blocks, Part 2

-   Deployment Rhythm by Example

-   Workshop Landing page

### CODE REVIEW

**Comment by billkable** [comment]
@BrianMMcClain Can you review [spell check run](https://github.com/vmware-tanzu/tanzu-dev-portal/pull/1135/checks?check_run_id=3226793303)?

The following are tagged as misspellings, yet these are valid words:

- Timeboxing
- Middleware
- codebases
- Kata

File: content/outcomes/application-development-rhythm/dev-flow.md [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+The *Continuous Delivery Pipeline* may involve various manual and/or
+automated steps for testing,
+verification,
+oversight compliance,
+staging,
+or perhaps a
+[secure pipeline towards production](https://www.thoughtworks.com/insights/articles/towards-a-secure-path-to-production).
+
+*Note*:
+There is much more to this topic that is outside the scope of this
```
Comment (joemoore) [inline content/outcomes/application-development-rhythm/dev-flow.md:109]: Consider providing links where the reader can learn more.

File: content/outcomes/application-development-rhythm/dev-flow.md [hunk 2]
```diff
@@ -0,0 +1,227 @@
...
+    or investigation if the problem/solution domain is not well known.
+    If the story is investigative,
+    the developer may use a
+    *[Spike](https://www.leadingagile.com/2016/09/whats-a-spike-who-should-enter-it-how-to-word-it/)*
+    as a way to timebox effort to better understand a problem and/or its
+    solution.
+1.  Iterate on a series of steps using one or more *feedback loops* to
+    achieve the outcome of the story.
+    Developers will spend much of their time writing code,
+    using methods such as *Test Driven Development*,
```
Comment (joemoore) [inline content/outcomes/application-development-rhythm/dev-flow.md:142]: Consider linking to TDD article in the Fundamentals course.

File: content/outcomes/application-development-rhythm/dev-building-blocks-1.md [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+The following is a good principle that is universal for software,
+regardless of the language or technology.
+
+***Code must be easy for any developer to read, adapt and extend.***
+
+NOTE:
+There are other, specific principles to support this broad one,
+but they are outside the scope of this article.
+
+## Pair Programming Practices
```
Comment (joemoore) [inline content/outcomes/application-development-rhythm/dev-building-blocks-1.md:61]: Are Pairing, TDD, etc sub-topics of "Coding and Architectural Design Principles and Practices"? If so consider making them 3rd-level headings with `### Heading`

File: content/outcomes/application-development-rhythm/dev-building-blocks-1.md [hunk 4]
```diff
@@ -0,0 +1,201 @@
...
+[Trunk-based development](https://trunkbaseddevelopment.com/) is the
+method of a team’s developers integrating on a *trunk* or *mainline* of
+a source control repository.
+
+Some examples of trunk naming in various source control systems:
+
+-   **Git**:
+    *main* is the preferred branch name.
+    Although the branch name of *master* may be found in older projects,
+    its use* *is deprecated.
```
Comment (joemoore) [inline content/outcomes/application-development-rhythm/dev-building-blocks-1.md:123]: Typo here?

File: content/outcomes/application-development-rhythm/dev-flow.md (hunk 2 above)
Comment (billkable) [reply content/outcomes/application-development-rhythm/dev-flow.md]: @joemoore thanks for feedback, fixed

File: content/outcomes/application-development-rhythm/dev-building-blocks-1.md (hunk 3 above)
Comment (billkable) [reply content/outcomes/application-development-rhythm/dev-building-blocks-1.md]: @joemoore thanks for feedback, fixed

File: content/outcomes/application-development-rhythm/dev-building-blocks-1.md (hunk 4 above)
Comment (billkable) [reply content/outcomes/application-development-rhythm/dev-building-blocks-1.md]: @joemoore yep.  thanks for catch, fixed

**Comment by joemoore** [comment]
LGTM 👍

**Pull request closed without merging**